providers/os/resources/windows.go

```diff
@@ -17,16 +20,35 @@ import (
 func (s *mqlWindows) computerInfo() (map[string]interface{}, error) {
 	conn := s.MqlRuntime.Connection.(shared.Connection)
 
-	cmd := windows.PSGetComputerInfo
```

Comment:
this function should already execute Get-ComputeInfo in case the other function fails. There must be something else wrong. I am trying to test